src/ucs/type/cpu_set.h

```diff
@@ -54,21 +54,20 @@ typedef struct {
         } \
     } while (0)
 
-#define UCS_CPU_ISSET(_cpu, _cpusetp) \
-    ({ \
-        size_t _cpu2 = (_cpu); \
-        (_cpu2 < (8 * sizeof(ucs_cpu_set_t))) ? \
-            (((((const ucs_cpu_mask_t *) ((_cpusetp)->ucs_bits))[UCS_CPUELT (_cpu2)] & \
-                UCS_CPUMASK (_cpu2))) != 0) : \
-            0; \
-     })
-
+static inline int ucs_cpu_is_set(size_t cpu2, const ucs_cpu_set_t *cpusetp)
```

Comment:
let's change to
`ucs_cpu_is_set(int cpu, const ucs_cpu_set_t *cpusetp)`
since the return value of `ucs_cpu_set_find_lcs` is int